dbcon/mysql/ha_mcs_pushdown.cpp

```diff
@@ -754,14 +743,14 @@ int ha_mcs_group_by_handler::end_scan()
 select_handler*
 create_columnstore_select_handler(THD* thd, SELECT_LEX* select_lex)
 {
-    ha_columnstore_select_handler* handler = NULL;
+    int select_handler_mode = (int) get_select_handler(thd);
```

Comment:
Could you declare it like this select_handler_mode_t select_handler_mode = get_select_handler(thd) ?

Reply:
Done.